web/src/components/OpeningHours.tsx

```diff
@@ -29,10 +31,23 @@ const TitleContainer = styled.div`
   ${helpers.adaptiveFontSize};
 `
 
+const OpeningContainer = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 8px;
+`
+
+const StyledIcon = styled(Icon)`
+  width: 18px;
+  height: 18px;
+  color: white;
```

Comment:
:x: This is making the note icon invisible, remove it
```suggestion
```

Reply:
Yup, in fact, the entire icon there is test code that I should have removed, thanks for catching that!